partners.json

```
@@ -1071,6 +1071,27 @@
       "picture": "/images/logos/partners/epci/grand-paris-sud.png",
       "height": 122,
       "width": 280
+    },
+    {
+      "name": "Communauté d’Agglomération du Puy-en-Velay",
+      "link": "https://geoportail.lepuyenvelay.fr/",
+      "infos": "La Communauté d’Agglomération du Puy-en-Velay met à disposition de ses communes un outil de gestion de leur Base Adresse Locale, permettant de faire remonter leurs voies et adresses au niveau national. Nous proposons également des formations et un accompagnement pour l’utilisation de cet outil, ainsi qu’un modèle de cahier des charges si vous souhaitez passer par un prestataire extérieur.",
+      "perimeter": "communes membres de la Communauté d’Agglomération du Puy-en-Velay (Haute-Loire, 43)",
+      "codeDepartement": [
+        "43"
+      ],
+      "echelon": 1,
+      "isPerimeterFrance": false,
+      "isCompany": false,
+      "services": [
+        "formation",
+        "accompagnement technique",
+        "réalisation de bases adresse locales",
```

```suggestion
        "réalisation de bases adresses locales",
```
Pour correspondre au tag

---

NB: La charte est dans un sous menu du menu à tiroir sans autre mention à la BAN